docs/rules/double-precision-literal.md

```diff
@@ -0,0 +1,29 @@
+# double-precision-literal (MOD002)
+This rule is unstable and in [preview](../preview.md). The `--preview` flag is required for use.
+
+## What it does
+Checks for floating point literals that using `d` or `D` as the exponentiation.
+
+## Why is this bad?
+Floating point literals using `d` or `D` as the exponentiation, such as `1.23d2`,
```

Comment:
Typo:
```suggestion
Floating point literals using `d` or `D` as the exponent, such as `1.23d2`,
```

Reply:
Thanks for the catch, fixed in https://github.com/PlasmaFAIR/fortitude/pull/390/commits/eb491cde94d93d9c604c0229fbe1d3af44db9d15